### PR TITLE
[mpg123] Upgrade to 1.26.4

### PR DESCRIPTION
**Pull request by evpobr**
Add next version In continuation of the pull request #16910.

### CODE REVIEW

File: versions/m-/mpg123.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "710ed26fdb0b54f6c10a5e1dbad8dbed163a1095",
+      "version-string": "1.26.4",
+      "port-version": 0
+    },
+    {
+      "git-tree": "0fe2ebdde4b38d6ec05197352c32db3dbcdc1df6",
+      "version-string": "1.26.4",
+      "port-version": 1
+    },
```
Comment (JonLiu1993) [inline versions/m-/mpg123.json:12]: @evpobr ，Please delete this duplicate "version", each version corresponds to only one git-tree.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]
Please add the .PC file processing function vcpkg_fixup_pkgconfig() under vcpkg_install_make() in portifle.cmake

**Comment by JonLiu1993** [comment]
@evpobr ,After you have modified everything, you should run the ./vcpkg x-add-version forge --overwrite-version command and then submit it again

**Comment by evpobr** [comment]
@JonLiu1993 , thanks, i know. I did everything I could without installing Vcpkg, which is impossible on this computer. I'll finish this pull request later tonight.

**Review by JonLiu1993 — APPROVED** [review]

File: ports/mpg123/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "mpg123",
-  "version-string": "1.26.3",
-  "port-version": 1,
+  "version-string": "1.26.4",
```
Comment (vicroms) [inline ports/mpg123/vcpkg.json:3]: Maybe change this to [`version` or `version-semver`](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md).

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/mpg123/vcpkg.json (hunk 2 above)
Comment (evpobr) [reply ports/mpg123/vcpkg.json:3]: I guess it could be "version".

**Pull request merged**